Runtime/Behaviors/MoveObjectBehavior.cs

```diff
@@ -39,6 +40,7 @@ public class EntityData : IBehaviorData
             /// <summary>
             /// Duration of the transition. If duration is equal or less than zero, target object movement is instantaneous.
             /// </summary>
+            [OptionalValue]
```

Comment:
Is it really optional? 🤔

Reply:
instant transition is fine for me :P

Reply:
Also would say 0 is fine

Reply:
Should this attribute be in a defined symbol?